tools/firtool/firtool.cpp

```diff
@@ -405,8 +406,10 @@ static LogicalResult processBuffer(
 
     // Run final IR mutations to clean it up after ExportVerilog and before
     // emitting the final MLIR.
-    if (!mlirOutFile.empty())
+    if (!mlirOutFile.empty()) {
       pm.addPass(firrtl::createFinalizeIRPass());
+      pm.addPass(om::createFreezePathsPass());
```

Comment:
Nit: Consider registering this pass so it's available for the `-mlir-print-ir-{before,after}` options.  Possibly could register all OM passes, at least for now?

Reply:
Added in a new commit.